src/OpenTelemetry.ResourceDetectors.Container/KubeApiConnector.cs

```diff
@@ -0,0 +1,48 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+#if !NETFRAMEWORK
```

Comment:
Probably we should drop Net Framework support all-together, as it is Windows only, while cgroup is very Linux specific.
Looks like K8 resolution with current implementation would also work only on Linux.
So, there is no code in Detector that may be useful in .Net Framework.